src/PixiEditor/ViewModels/Document/DocumentViewModel.cs

```diff
@@ -537,7 +538,7 @@ public OneOf<Error, Surface> TryRenderWholeImage(KeyFrameTime frameTime)
         IStructureMemberHandler? layerToExtractFrom = null)
     {
         layerToExtractFrom ??= SelectedStructureMember;
-        if (layerToExtractFrom is not LayerViewModel layerVm)
+        if (layerToExtractFrom is not ImageLayerNodeViewModel layerVm)
```

Comment:
The difference between this change and the previous is that `LayerViewModel` was a generic "Layer" view model, while you added specific layer type `ImageLayerNode`, It doesn't matter at this moment, but will when we have more layer types.